setup.py

```diff
@@ -18,7 +18,7 @@
 
 
 tests = [
-    "pytest",
+    "pytest<8",
```

Comment:
waiting for this it seems https://github.com/TvoroG/pytest-lazy-fixture/issues/65